docs/api/ShallowWrapper/render.md

```diff
@@ -30,5 +30,5 @@ function Bar() {
 ```jsx
 const wrapper = shallow(<Bar />);
 expect(wrapper.find('.in-foo')).to.have.lengthOf(0);
-expect(wrapper.find(Foo).render().find('.in-foo')).to.have.lengthOf(1);
+expect(wrapper.render().find('.in-foo')).to.have.lengthOf(1);
```

Comment:
same question here

Reply:
For this method, should I also use `.filter` to modify it in the same way as the ` .render ` in ReactWrapper ? Like this?
`expect(wrapper.find(Foo).render().filter(".in-foo")).to.Have.Length(1)`

Reply:
That seems great to me - ideally the examples in both the shallow and mount docs are as close to identical as possible, just like the test cases :-)